src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -614,19 +619,14 @@ public async Task<IActionResult> OptOutStandardVersion(OptOutStandardVersionView
             if (string.IsNullOrEmpty(model.Version))
                 throw new ArgumentException($"Value of {nameof(model.Version)} cannot be null or empty");
 
-            if (!ModelState.IsValid)
-            {
-                return RedirectToRoute(OptOutStandardVersionRouteGet, new { referenceNumber = model.StandardReference, version = model.Version });
-            }
-
             var contactId = await GetUserId();
             var epaOrgId = GetEpaOrgIdFromClaim();
 
             var approvedVersions = await _standardVersionApiClient.GetEpaoRegisteredStandardVersions(epaOrgId, model.StandardReference);
 
             if (approvedVersions.FirstOrDefault(p => p.Version.Equals(model.Version, StringComparison.InvariantCultureIgnoreCase)) == null)
             {
-                throw new NotFoundException($"Unable to opt out of StandardReference {model.StandardReference} organisation {epaOrgId} does not assesses this standard version");
+                return View(model);
```

Comment:
This check should not be required any more?

Reply:
Removed